### PR TITLE
[Bug 1176714] rename "post reply" to "quote"

### PR DESCRIPTION
**Pull request by philipp-sumo**
...under the initial question text to avoid misconceptions what this button actually does.
also change styling from a button to a link and add a padding to its right in order for it to fit better into the rest of the page, as the "quote"-links under replies are styled that way as well.

### CODE REVIEW

File: kitsune/questions/templates/questions/question_details.html [hunk 1]
```diff
@@ -166,7 +166,7 @@ <h2 class="summary" itemprop="headline">{{ question.title }}</h2>
               </form>
             </div>
           {% endif %}
-          <a class="btn quoted-reply" href="#question-reply" data-content-id="question-{{ question.id }}">{{ _('Post a Reply') }}</a>
+          <a class="reply quoted-reply" href="#question-reply" data-content-id="question-{{ question.id }}">{{ _('Quote') }}</a>
```
Comment (rehandalal) [inline kitsune/questions/templates/questions/question_details.html]: you don't need to add the class `reply` here.

Comment (philipp-sumo) [inline kitsune/questions/templates/questions/question_details.html]: thanks for vetting my attempts.

File: kitsune/sumo/static/sumo/less/questions.less [hunk 2]
```diff
@@ -317,6 +317,7 @@ h2 {
 
   .question-tools {
     margin: 0 0 20px 130px;
+    padding: 0 20px;
```
Comment (rehandalal) [inline kitsune/sumo/static/sumo/less/questions.less]: You probably want to make this:
```css
padding: 0 20px 0 0;
```

**Comment by rehandalal** [comment]
almost there!

File: kitsune/sumo/static/sumo/less/questions.less (hunk 2 above)
Comment (philipp-sumo) [inline kitsune/sumo/static/sumo/less/questions.less]: i've borrowed this from our current .answer-tools styling. should i change it in line 127 as well?

Comment (rehandalal) [inline kitsune/sumo/static/sumo/less/questions.less]: no please only change it here.

File: kitsune/sumo/static/sumo/less/questions.less [hunk 3]
```diff
@@ -126,7 +126,7 @@ h2 {
 
   .answer-tools {
     margin: 0 0 20px 130px;
-    padding: 0 20px;
+    padding: 0 20px 0 0;
```
Comment (rehandalal) [inline kitsune/sumo/static/sumo/less/questions.less]: please don't change this line.

File: kitsune/sumo/static/sumo/less/questions.less (hunk 2 above)
Comment (rehandalal) [inline kitsune/sumo/static/sumo/less/questions.less]: you didn't actually change it here.

File: kitsune/sumo/static/sumo/less/questions.less (hunk 3 above)
Comment (philipp-sumo) [inline kitsune/sumo/static/sumo/less/questions.less]: sorry!

**Comment by rehandalal** [comment]
Thanks! r+

**Pull request merged**